core/tauri/src/api/path.rs

```diff
@@ -57,16 +57,26 @@ pub enum BaseDirectory {
   Video,
   /// The Resource directory.
   Resource,
-  /// The default App config directory.
-  /// Resolves to [`BaseDirectory::Config`].
+  /// Alias of [`BaseDirectory::AppConfig`]
   App,
-  /// The Log directory.
-  /// Resolves to `BaseDirectory::Home/Library/Logs/{bundle_identifier}` on macOS
-  /// and `BaseDirectory::Config/{bundle_identifier}/logs` on linux and windows.
+  /// Alias of [`BaseDirectory::AppLog`]
```

Comment:
Keep the doc comments and add a deprecation note https://doc.rust-lang.org/reference/attributes/diagnostics.html?highlight=deprecate#the-deprecated-attribute saying it will be removed in v2 and suggest the new variant